stdlib/http/src/main/ballerina/http/resiliency/load_balance_client_endpoint.bal

```diff
@@ -217,35 +221,36 @@ public type LoadBalanceActionError error<string, LoadBalanceActionErrorData>;
 // Performs execute action of the Load Balance connector. extract the corresponding http integer value representation
 // of the http verb and invokes the perform action method.
 function performLoadBalanceExecuteAction(LoadBalanceClient lb, string path, Request request,
-                                         string httpVerb) returns @tainted Response|error {
+                                         string httpVerb) returns Response|ClientError {
     HttpOperation connectorAction = extractHttpOperation(httpVerb);
     if (connectorAction != HTTP_NONE) {
         return performLoadBalanceAction(lb, path, request, connectorAction);
     } else {
-        error httpActionErr = error("Unsupported connector action received.");
-        return httpActionErr;
+        string message = "Load balancer client not supported for connector action: " + httpVerb;
```

Comment:
What's meant by the connector action? It is possible to rename that with more user friendly manner wording?

Reply:
Will update